packages/discovery-provider/plugins/notifications/src/listener.ts

```diff
@@ -38,6 +38,8 @@ export class Listener {
     logger.info('did connect')
 
     this.client.on('notification', async (msg: Notification) => {
+      // Only process events from notification table
+      if (msg.channel !== 'notification') return
```

Comment:
interesting

Reply:
yeah I got confused because `'notification'` sounded like it was specific to the notificaitons table event, but it's actually the category from the psql client. `msg.channel` is set by `pg_notify(TG_TABLE_NAME` in our trigger